frontend/projects/upgrade/src/app/features/dashboard/home/components/metrics/metrics.component.ts

```diff
@@ -286,7 +299,7 @@ export class MonitoredMetricsComponent implements OnInit, OnChanges, OnDestroy {
     operationType = null,
     compareFn = null,
     compareValue = null,
-    repeatedMeasure = REPEATED_MEASURE.mostRecent
+    repeatedMeasure = null
```

Comment:
Actually, this is the line that is causing @zackcl 's global metric issue. Should repeatedMeasure have a default value if null is unacceptable?

Reply:
Thanks @jreddig. Yeah, I just checked, Zack has updated this in his [commit](https://github.com/CarnegieLearningWeb/UpGrade/pull/865/commits/490351ff173f5114f21b7a87a5d1773d0b75db0b). Actually `repeatedMeasure` has `MOST RECENT` as default value and is not nullable:true in the model. We need to have `MOST RECENT` by default in the global metrics as well. I will revert this change.
![Uploading Screenshot 2023-06-30 at 4.02.58 PM.png…]()